### PR TITLE
Add response codes to the report endpoints

### PR DESCRIPTION
**Pull request by jbartece**

### CODE REVIEW

File: reports-rest/.tern-project [hunk 1]
```diff
@@ -0,0 +1 @@
+{"libs":["browser","ecma5"],"ide":{}}
```
Comment (thescouser89) [inline reports-rest/.tern-project]: Is that intentional?

File: reports-rest/src/main/java/org/jboss/da/rest/reports/Reports.java [hunk 2]
```diff
@@ -17,13 +17,18 @@
 import javax.ws.rs.POST;
 import javax.ws.rs.Path;
 import javax.ws.rs.Produces;
+import javax.ws.rs.WebApplicationException;
```
Comment (thescouser89) [inline reports-rest/src/main/java/org/jboss/da/rest/reports/Reports.java]: Unused import statement

File: reports-rest/.tern-project (hunk 1 above)
Comment (jbartece) [inline reports-rest/.tern-project]: No. Sorry for this stupid issue

**Comment by thescouser89** [comment]
:+1:

**Pull request merged**